poster/poster.tex

```diff
@@ -181,20 +181,22 @@ \section*{Introduction}
 be opened in the whole Ginga session, whereas a local plugin is
 associated with the channel it is started from: one instance can be
 opened per channel and different instances can be configured separately
-in the same Ginga session. 
-Plugins are currently not available when Ginga is used as part of Jupyter
```

Comment:
I don't think these two lines are needed. 1) the poster is about plugins for the reference viewer, so mentioning the notebook viewer is not necessary or relevant, and 2) nowhere are we talking about remote control of the viewer.